packages/components/src/__actions__/LinkButton/v3/_docs/LinkButton--api-specification.mdx

```diff
@@ -0,0 +1,252 @@
+import { Canvas, Meta, Controls, ArgTypes, DocsStory } from "@storybook/blocks"
+import { ResourceLinks, KAIOInstallation } from "~storybook/components"
+import { LinkTo } from "../../../../../../../docs/components/LinkTo"
+import * as exampleStories from "./LinkButton.doc.stories"
+
+<Meta title="Actions/LinkButton/LinkButton (v3)/API Specification" />
+
+# LinkButton API Specification (v3)
+
+Updated Nov 26, 2024
+
+<ResourceLinks
+  sourceCode="https://github.com/cultureamp/kaizen-design-system/tree/main/packages/components/src/__actions__/LinkButton/v3"
+  figma="https://www.figma.com/design/eZKEE5kXbEMY3lx84oz8iN/%F0%9F%92%9C-Heart-UI-Kit?node-id=1929-17364"
+  designGuidelines="/?path=/docs/actions-linkbutton-linkbutton-v3-usage-guidelines--docs"
+/>
+
+<KAIOInstallation exportNames={["LinkButton" ]} family="actions" version="3" />
+
+## Overview
+
+`LinkButton` allows users to navigate to another page or resource. It shares the same visual styles and interaction states as the <LinkTo pageId="actions-button-button-v3-usage-guidelines--docs">Button</LinkTo> component, but is intended for navigational purposes and downloading documents.
+
+The following example and table showcases the essential props that enable the core functionality of `LinkButton`. For the remaining suite of API options refer to [this section](#additional-api-options).
+
+<Canvas of={exampleStories.Playground} />
+
+<Controls of={exampleStories.Playground} include={["className", "children", "href", "target", "download", "onPress", "routerOptions", "hasHiddenLabel",  "size", "variant", "icon", "iconPosition", "isFullWidth", "isDisabled" ]} />
+
+## API
+
+This is built on top of [React Aria's Link component](https://react-spectrum.adobe.com/react-aria/Link.html) and is the counterpart to the <LinkTo pageId="actions-button-button-v3-api-specification--docs">Kaizen Button</LinkTo>, handling icons, variants and sizes in the same way. It provides a semantic wrapper for navigational buttons and allows for native `href` navigation and client side routing with [additional configuration](#client-side-routing).
+
+### Navigation and native anchor attributes
+
+Out of the box, the `LinkButton` offers majority of the native behavior and functionality on the `anchor` tag. `href` will trigger new page loads, `download` will download the referenced document, and `target` can be used to open links in new tabs or windows.
+
+<Canvas of={exampleStories.DownloadIconButton} />
+
+While client side routing is possible, the `LinkButton` is agnostic to the routing technology chosen. Refer to our [general set up guide here](#client-side-routing) to get started with client side routing.
```

Comment:
Or reword if you'd like to ensure the "guide" part is part of the link.
```suggestion
While client side routing is possible, the `LinkButton` is agnostic to the routing technology chosen. Refer to our general set up guide to get started with client side routing((#client-side-routing).
```

Reply:
looks like that might be missing the `[]`, how does:
```
While client side routing is possible, the `LinkButton` is agnostic to the routing technology chosen. Refer to our general set up guide to get started with [client side routing](#client-side-routing).
```
look?